packages/wdio-reporter/src/index.js

```diff
@@ -31,6 +31,14 @@ export default class WDIOReporter extends EventEmitter {
             failures: 0
         }
 
+        const outputDir = this.options.outputDir
+        if (outputDir) {
+            // make sure report directory exists
+            if (!fs.existsSync(outputDir)){
+                fs.mkdirSync(outputDir)
```

Comment:
Does this create the directory recursively? What happen if the directory in which the directory should be set does not exist?

Reply:
That's a really good point. No, don't think it does handle recursive creation. Let me add a test and support for that.